src/batcher.ts

```diff
@@ -203,11 +182,7 @@ export class Batcher {
               resolve(this.waitForCompletion(newBatch, batchesCompleted, overallInfo, i + 1, batches.length, wait));
             }
 
-            newBatch.execute(batch, (err) => {
-              if (err) {
-                reject(err);
-              }
-            });
+            void newBatch.execute(batch);
```

Comment:
I'm really not sure about this.  I think it might be an error in the jsforce typings, or something that changes from whenever this was created

it's worth digging around in your IDE on this to see what's happening with types.

`execute(input?: string | Record[] | Readable): this;`